apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/RequestOptions.cs

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using System;
+
+namespace Google.Cloud.Spanner.Data
+{
+    /// <summary>
+    /// Immutable class representing request options.
+    /// </summary>
+    internal class RequestOptions : IEquatable<RequestOptions>
+    {
+        /// <summary>
+        /// The request tag configured in the options.
+        /// </summary>
+        internal string RequestTag { get => Proto.RequestTag; }
```

Comment:
This can be simplified to

```csharp
internal string RequestTag => Proto.RequestTag;
```

(Ditto for TransactionTag.)

I'm not sure why GitHub isn't offering me the "suggestion" option...